src/ui/screens/tipsAndTricks/ViewTipsAndTricks.qml

```diff
@@ -40,21 +40,42 @@ VPNViewBase {
         }
 
         ObjectModel {
-            id: tipsSections
+            id: guidesSections
 
+            // Quick tips
             Item {
                 property string title: VPNl18n.TipsAndTricksQuickTipsTitle
                 property string description: VPNl18n.TipsAndTricksQuickTipsDescription
                 property var filter: (addon) => !addon.advanced && !addon.highlighted
             }
 
+            // Advanced tips
             Item {
                 property string title: VPNl18n.TipsAndTricksAdvancedTipsTitle
                 property string description: VPNl18n.TipsAndTricksAdvancedTipsDescription
                 property var filter: (addon) => addon.advanced && !addon.highlighted
             }
         }
 
+        ObjectModel {
+            id: tutorialsSections
+
+            // Highlighted tutorials
+            Item {
+                property string title: VPNl18n.TipsAndTricksTutorialsTitle
+                property string description: VPNl18n.TipsAndTricksTutorialsDescription
+                property var filter: (addon) => addon.highlighted
+                property var featureTourCardVisible: true
+            }
+
+            // More tutorials
+            Item {
+                property string title: VPNl18n.TipsAndTricksMoreTutorialsTitle
+                property string description: VPNl18n.TipsAndTricksMoreTutorialsDescription
+                property var filter: (addon) => !addon.highlighted
+            }
+        }
+
         stackContent: [
             // All
             VPNFlickable {
```

Comment:
Instead of flickables, we can use `VPNViewBase` which is essentially a flickable that factors in the nav bar so we don't have to set a weird flickContentHeight